05_stack_and_queue/src/sortStation.c

```diff
@@ -0,0 +1,119 @@
+#include "stack.h"
+
+#include <ctype.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+
+int getPriority(char operation)
+{
+    if (operation == '*' || operation == '/') {
+        return 2;
+    } else if (operation == '+' || operation == '-') {
+        return 1;
+    }
+    return 0;
+}
+
+int sortStation(char* str, char* res)
+{
+    unsigned lastUsedResIndex = 0;
+    Stack* stack = newStack();
+    unsigned long len = strlen(str);
```

Comment:
Minor: 
```suggestion
    size_t len = strlen(str);
```

Ну это совсем придирка, просто `strlen()` отдает `size_t`, наверняка есть платформы, где `unsigned long` будет 32 бита, а `size_t` --- 64.